integration/setup/bastion.go

```diff
@@ -120,7 +120,7 @@ func CreatePublicIP(ctx context.Context, location, groupName string, ipClient ne
 func CreateVM(ctx context.Context, location, groupName, sshKeyData string, nic network.Interface, config Config) error {
 	vmName := "bastionE2EVirtualMachine"
 	username := "e2e"
-	config.Logger.LogCtx(ctx, "message", "Creating e2e bastion instance", "vmName", vmName)
+	config.Logger.LogCtx(ctx, "message", "Creating e2e bastion instance", "vmName", vmName) // nolint: errcheck
```

Comment:
Why do we need all those `// nolint: errcheck`? Is this something we can fix in the micrologger?

Reply:
https://gigantic.slack.com/archives/C0251MXL5/p1584003641116900

Reply:
I'm not sure a logger should return an error.
Until a decision is made, I believe it's fine to ignore the lint errors in log calls.